cypress/views/bucket-policy.ts

```diff
@@ -0,0 +1,64 @@
+import { DATA_FEDERATION_NAMESPACE } from '../constants/common';
+import { Providers, TEST_DATA_SOURCE } from '../constants/tests';
+import { createDataSource } from './data-resource';
+
+export const createBucket = (
```

Comment:
Since it will be part of BPCommon, we can name the function `createUsingSingleDS` so that in future we can add methods for creating BP using multi DS